### DISCUSSION TITLE
Keycloak backwards compatibility policy vs Keycloak 26

### DISCUSSION THREAD

**Discussion by tbroyer** [keycloak/keycloak#33990] [Q&A] [answered]
It's not clear from [the blog post](https://www.keycloak.org/2024/10/release-updates.html) whether this applies from 26 onwards (so v27 will be in 2–3 years), or from v27 onwards? (so 27 in a few months, then 28 2–3 years after that)

**Comment by stianst** [comment, marked as answer]
Keycloak 27 will be in 2 years time. In the meantime there will be 26.y minor releases.
